applications_tests/gls_nitsche_navier_stokes_22/nitsche-heating_moving.prm

```diff
@@ -0,0 +1,130 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set time end                = 5
+  set time step               = 1
+  set output name             = hydrostat_nitsche-HT
+  set output frequency        = 1
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+subsection multiphysics
+  set heat transfer = true
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = nodal
+    subsection temperature
+            set Function expression = 0
+    end
```

Comment:
Aren't these the default values?

Reply:
Yes it is! But I thought it would be clearer to specify it, as we are monitoring the temperature evolution.

Reply:
Fair enough!